dev/github/run_checks.sh

```diff
@@ -28,10 +30,10 @@ fi
 
 export MAVEN_OPTS="-Dorg.slf4j.simpleLogger.showDateTime=true -Dorg.slf4j.simpleLogger.dateTimeFormat=HH:mm:ss.SSS"
 
-# Always use java 8 to compile the source code
+# Always use java 11 to compile the source code
```

Comment:
I think we had this line of command to note that although we are running tests in java 11, we still built with java 8. Now if everything is 11, this comment doesn't serve any purpose.

Reply:
we may introduce a java17 build, so i left these extraneous comments and no-op settings

Reply:
ok